test/upload_test.js

```diff
@@ -138,6 +138,27 @@ describe('S3 Webpack Upload', function() {
       .then(randomFileBody => assert.match(randomFileBody, testHelpers.S3_ERROR_REGEX, 'random file exists'))
   })
 
+  it('starts a index change', function() {
```

Comment:
As you mentioned this test is pretty useless as it's not even checking the index file you just uploaded!